README.md

```diff
@@ -124,6 +124,13 @@ A **Finding Group** is a collection of findings that are scored in the same way
 ### Page Template
 A **Page Template** lets you customize report background images and footers.  You can set one **Page Template** as the default, and it will be applied globally unless overridden at the **Engagement** or **Report** level.
 
+## Markdown placeholders
+You can automatically insert client-specific information such as the client name, URL, e-mail, etc. in your reports, by inserting
+`{Client<field>}` in the text. This is particularly useful for report templates.
+
+For example, if you want to refer to the client in your executive summary, you can insert `{ClientName}` in the text. For a specific
+list of fields you can insert, or to insert more, refer to [the markdown.py file](writehat/lib/markdown.py)
+
```

Comment:
```suggestion
Finally, please note that you can also use the editor's 🔗 icon to select these placeholders (and more!).
```
@thejohnbrown regarding your comment, what do you think about adding this snippet? Do you think it's enough or would a screenshot like the one you suggested be better?